src/util/notary-util.js

```diff
@@ -1,8 +1,16 @@
 import axios from 'axios'
-import { ETH_NOTARY_URL, BTC_NOTARY_URL } from '@/data/urls'
+// import { ETH_NOTARY_URL, BTC_NOTARY_URL } from '@/data/urls'
 
-let axiosNotary = axios.create({
-  baseURL: ETH_NOTARY_URL
+const axiosNotaryRegistration = axios.create({
+  baseURL: ''
+})
+
+const axiosNotaryETH = axios.create({
```

Comment:
I think we should change notary to registration word, because it's only registration service on this address

Reply:
No, we also use this address for requesting amount of free relays.